src/assets/translations/fr/main.json

```diff
@@ -2133,14 +2141,52 @@
     }
   },
   "lending": {
+    "lending": "Prêts",
+    "lendingBody": "Une liste de tous les pools pouvant être empruntés à taux zéro et sans liquidation.",
+    "collateralValue": "Valeur du collatéral",
+    "debtValue": "Valeur de la dette",
+    "loanToValue": "Rapport prêt-valeur",
+    "availablePools": "Pools disponibles",
+    "yourLoans": "Vos prêts",
+    "pool": "Pool",
+    "poolDepth": "Profondeur du pool",
+    "totalDebtBalance": "Solde total des dettes",
+    "totalCollateral": "Totale des collatéraux",
+    "estCollateralizationRatio": "Est. du ratio de collatéralisation",
+    "collateralizationRatio": "Ratio de collatéralisation",
+    "totalBorrowers": "Total d'emprunteurs",
+    "myLoanInformation": "Informations sur mon prêt",
+    "loanInformation": "Résumé du prêt",
+    "poolInformation": "Informations sur le pool",
+    "collateralBalance": "Solde du collatéral",
+    "debtBalance": "Solde de la dette",
+    "repaymentLock": "Verrou du remboursement",
```

Comment:
Note related to your PR but just FYI, I found this to be quite confusing in English as well, wondering if `Repayment Unlock` would make more things more clear as a user

Reply:
I think Product had this discussion and settled on this wording... but it's true that depending on how you think about it, it can be interpreted differently. If it ever changes in English we'll update translations anyways :D Thanks for the review !